library/rendering/entity_rendering/src/testmod/java/org/quiltmc/qsl/rendering/entity/test/QuiltArmorMaterial.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Just a complaint about Java in general: dear god there's a heckton of boilerplate